src/extensions/mountainview/MountainView/MountainView.tsx

```diff
@@ -14,24 +14,32 @@ type AddViewAction = {
     type: 'AddView'
     pluginType: ViewPluginType
     plugin: ViewPlugin
+    area: 'north' | 'south' | ''
```

Comment:
The two areas are called 'north' and 'south'. The '' means that it should be assigned to whatever is the current active area.